fe/fe-core/src/main/java/com/starrocks/lake/compaction/SimplifyPartitionStatistics.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package com.starrocks.lake.compaction;
+
+import javax.validation.constraints.NotNull;
+
+// simplified version of `PartitionStatistics`, only contains necessary copied info for compaction sorter
+public class SimplifyPartitionStatistics {
```

Comment:
1. Should define an interface `IPartitionStatistics`, with the three methods required. Both `PartitionStatistics` and this `SimplifyPartitionStatistic` implements the interface.

The `Selector` and the `Sorter` should implement algorithm based on `Collection<T implements IPartitionStatistics>`

2. maybe rename this class to `PartitionStatisticsSnapshot`

3. this instance can be created directly from `PartitionStatistics` object, e.g..  

```java

public SimplifyPartitionStatistics getStatsSnapshot() {
    return new SimplifyPartitionStatistics(this);
}